packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,18 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the politeness setting of aria live in Flutter web.
```

Comment:
Consider explaining the effect of this setting without repeating the words from the name of the enum. Words "politeness", "setting", "aria", and "live" appear in the name (although below I recommend that we remove "aria" and "live").